src/segment-loader.js

```diff
@@ -1328,4 +1562,144 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     this.segmentMetadataTrack_.addCue(cue);
   }
+
+  // TODO: documentation
+  maxBufferedBytes(start, end) {
```

Comment:
I'm not sure this name is quite accurate in terms of what it returns when you pass in arguments. Would something like `getCurrentlyBufferedBytes` be more accurate?

Reply:
It would not. `getBufferedBytesUpperBound` would be, though.